### PR TITLE
Added container-context flag to build command

### PR DESCRIPTION
**Pull request by michaelf-stratoscale**
The flag is optional and by default it will take the path of the dockerfile.

skipper build --help
Usage: skipper build [OPTIONS] [IMAGE...]

  Build a container

Options:
  --container-context TEXT  Container context path
  --help                    Show this message and exit.

### CODE REVIEW

**Comment by shay-stratoscale** [comment]
don't you want to add this option to skipper.yaml ?

**Comment by shay-stratoscale** [comment]
please note that clean-build job will not use this argument

**Comment by michaelf-stratoscale** [comment]
1. it's not a must but i can add the support, what need to done? 
2. I don't have a problem with clean build it's just an optional param.

**Comment by eran-stratoscale** [comment]
Since skipper is meant to "Easily dockerize your Git repository", As long as the context is in the project I'm fine with it.
Please adjust the test so that the context will be inside the project.
And add a documentation in the readme.

File: tests/test_cli.py [hunk 1]
```diff
@@ -168,6 +168,29 @@ def test_build_existing_image(self, skipper_runner_run_mock, *args):
...
+            global_params=self.global_params,
+            subcmd='build',
+            subcmd_params=build_params
+        )
+        expected_command = [
+            'docker',
+            'build',
+            '-f', '/home/user/work/project/Dockerfile.image1',
+            '-t', 'image1:1234567',
+            '/home/user/work'
```
Comment (eran-stratoscale) [inline tests/test_cli.py]: Change to path inside the project

File: skipper/cli.py [hunk 2]
```diff
@@ -36,8 +36,9 @@ def cli(ctx, registry, build_container_image, build_container_tag, build_contain
 
 @cli.command()
 @click.argument('images_to_build', nargs=-1, metavar='[IMAGE...]')
+@click.option('--container-context', help='Container context path', default=None)
```
Comment (eran-stratoscale) [inline skipper/cli.py]: Add to readme.

File: tests/test_cli.py (hunk 1 above)
Comment (michaelf-stratoscale) [inline tests/test_cli.py]: the point was to show a different path, but i'll change it.

**Comment by michaelf-stratoscale** [comment]
@eran-stratoscale @shay-stratoscale @ravidbro added another revision

**Pull request merged**